src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -131,17 +131,25 @@
                       :render-fn       render-transaction
                       :empty-component (empty-text (i18n/label :t/transactions-unsigned-empty))}]]))
 
-(defn- unsigned-transactions-title [unsigned-transactions-count]
-  (str (i18n/label :t/transactions-unsigned)
-       (if (pos? unsigned-transactions-count) (str " " unsigned-transactions-count))))
+(defn unsigned-transactions-title [{:keys [uppercase? style]} title]
```

Comment:
I don't think `uppercase?` and `style` should be managed by wrapping component as those are specific to this particular implementation.
If the idea was to share behavior between default `title-fn` and this one I'd rather we use functions and duplicate the definition of properties.